en/best-practices.md

```diff
@@ -62,9 +73,11 @@ __Examples:__
 | | By default, `stop_name` should not contain generic or redundant words like “Station” or “Stop”, but some edge cases are allowed.<ul><li>When it is actually part of the name (Union Station, Central Station<li>When the `stop_name` is too generic (such as if it is the name of the city). “Station”, “Terminal”, or other words make the meaning clear.</ul> |
 | `stop_lat` & `stop_lon` | Stop locations should be as accurate possible. Stop locations should have an error of __no more__ than four meters when compared to the actual stop position. |
 | | Stop locations should be placed very near to the pedestrian right of way where a passenger will board (i.e. correct side of the street). |
+| | Two stop or station locations should not be closer than would be physically possible in the real world, assuming accurate locations.  If two stops or stations are significantly close, it may indicate incorrect stop or station modelling. |
 | | If a stop location is shared across separate data feeds (i.e. two agencies use exactly the same stop / boarding facility), indicate the stop is shared by using the exact same `stop_lat` and `stop_lon` for both stops. |
```

Comment:
Does this apply to the station also, and if so, is it worth mentioning it just like the above statement?